backend/dataall/modules/dataset_sharing/services/share_item_service.py

```diff
@@ -101,16 +102,10 @@ def revoke_items_share_object(uri, revoked_uris):
 
             share_sm.update_state(session, share, new_share_state)
 
-            if share.groupUri != dataset.SamlAdminGroupName:
-                revoke_table_items = ShareObjectRepository.find_all_share_items(
-                    session, uri, ShareableType.Table.value, [ShareItemStatus.Revoke_Approved.value]
-                )
-                for item in revoke_table_items:
-                    ResourcePolicyService.delete_resource_policy(
-                        session=session,
-                        group=share.groupUri,
-                        resource_uri=item.itemUri,
-                    )
+            if share.groupUri != dataset.SamlAdminGroupName and share.principalType == PrincipalType.Group.value:
```

Comment:
something to consider is should this permission removal be handled here or AFTER revoke is successful (i.e. at the end of share manager task)? 

For the case when revoke does fail and in a "limbo" state where in this case user would not be able to get the folder or table from data.all UI but the share could potentially still be functional

... something to consider - can be a separate PR

Reply:
makes sense think this is fine as is